googletest/test/googletest-param-test-test.cc

```diff
@@ -490,16 +490,15 @@ TEST(CombineTest, CombineWithMaxNumberOfParameters) {
 class NonDefaultConstructAssignString {
  public:
   NonDefaultConstructAssignString(const std::string& s) : str_(s) {}
+  NonDefaultConstructAssignString() = delete;
+  NonDefaultConstructAssignString(const NonDefaultConstructAssignString&) = default;
+  NonDefaultConstructAssignString& operator=(const NonDefaultConstructAssignString&) = delete;
+  ~NonDefaultConstructAssignString() = default;
```

Comment:
What is the reason for explicitly declaring the destructor?

Reply:
No reason. I'm just in the habit of declaring all the special members — at least the Big 3 or the Big 5, but not, like, 2 of the 3, or 4 of the 5. I can remove the destructor declaration here if you want.